src/main/java/com/onarandombox/MultiverseCore/configuration/node/NodeGroup.java

```diff
@@ -1,15 +1,11 @@
 package com.onarandombox.MultiverseCore.configuration.node;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.Map;
-
 import io.github.townyadvanced.commentedconfiguration.setting.CommentedNode;
 import io.vavr.control.Option;
 import org.jetbrains.annotations.NotNull;
 
+import java.util.*;
```

Comment:
Don't use wildcard imports, intellij has a config for it. I usually just set like to 9999 imports before using wildcards lol